packages/bpk-component-scrollable-calendar/stories.js

```diff
@@ -20,8 +20,14 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
-import BpkScrollableCalendar from '.';
+import { BpkScrollableCalendarDate } from './index';
 
-storiesOf('bpk-component-scrollable-calendar', module).add('Default', () => (
-  <BpkScrollableCalendar />
```

Comment:
Why remove the default story? This should remain unless I am missing something...